cmp/ag-grid/AgGridModel.js

```diff
@@ -486,7 +525,7 @@ export class AgGridModel {
     }
 
     getPivotColumnId(column) {
-        return [...column.colDef.pivotKeys, column.colDef.pivotValueColumn.colId];
+        return [column.colDef.pivotKeys, column.colDef.pivotValueColumn.colId];
```

Comment:
why no longer spreading these?

Reply:
The id retrieved here is saved to state, and then used on line 271 by the 
agColumnApi.getSecondaryPivotColumn method, which has a signature that expects the first arg to be an array.
https://www.ag-grid.com/javascript-grid-pivoting/#looking-up-secondary-columns